README.md

```diff
@@ -392,13 +392,24 @@ Below is a list of all the settings that were added in ddnet-insta.
 + `sv_score_diff` Difference between the team-scores before a team can win (only foot)
 + `sv_sudden_death_score_diff` Difference between the team-scores before a team can win in sudden death (only foot)
 + `sv_suicide_penalty` Indicates if score is decremented on suicide
++ `sv_hammer_freeze` The amount of ticks of freeze inflicted when hitting an alive player. (0 to disable)
++ `sv_bombtag_bombs_per_player` The amount of bombs that should spawn per X alive players, 1 for everyone except one.
++ `sv_bombtag_score_for_winning` The amount of score given for the player who wins the round.
```

Comment:
Let's remove the `sv_bombtag_score_for_winning` config. It does not fit the ddnet-insta scoring style. There is the `points` column which should be incremented for kills and other in game actions. And there is `win_points` you get for winning, not regular points. And here we should do it like in zCatch more win points for more players. But that can be done in a follow up pr. For now we can just go with 0 points and 1 win_points for a win.